lib/RestClient.ts

```diff
@@ -174,9 +174,12 @@ export class RestClient {
 
     private async _processResponse<T>(res: httpm.HttpClientResponse, options: IRequestOptions): Promise<IRestResponse<T>> {
         return new Promise<IRestResponse<T>>(async (resolve, reject) => {
-            let rres: IRestResponse<T> = <IRestResponse<T>>{};
-            let statusCode: number = res.message.statusCode;
-            rres.statusCode = statusCode;
+            const statusCode: number = res.message.statusCode;
+
+            const rres: IRestResponse<T> = {
```

Comment:
Can we rename this to response or something more readable?

Reply:
I retained the original variable name, but yes, this should be renamed.